### ISSUE TITLE
Search Processes not working correctly

### ISSUE THREAD

**Issue by kumar007git** [adirh3/Fluent-Search#584]
**Describe the bug**
I am searching for processes and when the processes search box is shown and I type the name of a process the process is not showing in the search result even though the process exists in task manager. Also, only a handful of processes are showing the processes search when in fact many processes are running in task manager.

**To Reproduce**
1. Call up Processes search box
2. Search for a process
3. Only a handful of processes are shown compared to the number of running processes in task manager

**Expected behavior**
Should show all running processes in the task manager

**Desktop (please complete the following information):**
 - Windows 10 Version: 21H2
 - Fluent Search Version 0.9.91.50

**Comment by adirh3** [comment]
@kumar007git, it's confusing but processes search will show only processes with a Window.
Does that process have a window (UI)?

If it's a UWP process (Microsoft Store app), please update Fluent Search to the latest nightly version (0.9.91.66), as it fixes some issues with UWP processes.

**Comment by kumar007git** [comment]
Thanks. yes, the process has a window but it is minimized to system tray. It does not work even though I restore the app window from system tray. It is not a UWP or a microsoft store app.

**Comment by adirh3** [comment]
> Thanks. yes, the process has a window but it is minimized to system tray. It does not work even though I restore the app window from system tray. It is not a UWP or a microsoft store app.

I see if it's minimized to system tray Fluent Search will not show in process search.
If you want to close that process, then you can install the "Kill process" plugin which lets you close any process.

**Comment by kumar007git** [comment]
Yup, if minimized to system tray then it's not showing but the process is still running in task manager. I guess it is the way Fluent Search enumerates processes but it would be nice to be able to also search for apps minimized to system tray. Can you provide this capability?

I don't want to kill the app, I just want to show the window of that app so I can work in it.

**Comment by adirh3** [comment]
> Yup, if minimized to system tray then it's not showing but the process is still running in task manager. I guess it is the way Fluent Search enumerates processes but it would be nice to be able to also search for apps minimized to system tray. Can you provide this capability?
> 
> I don't want to kill the app, I just want to show the window of that app so I can work in it.

Sorry for late reply, if the app is minimized to system there is no way to bring it up in 100% of the cases.
Opening the `App` result should bring it app in most cases, depends on the app implementation.

**Comment by gitthangbaby** [comment]
i'd also like all processes to be listed. Hidden processes listed below the UI processes. It'd be nice to ShiftDel stuck processes.

**Comment by adirh3** [comment]
> i'd also like all processes to be listed. Hidden processes listed below the UI processes. It'd be nice to ShiftDel stuck processes.

Have you tried the Kill processes plug-in? It's very useful for this scenario
